rfcs/019-platform_reliability/observability.md

```diff
@@ -0,0 +1,39 @@
+# Observability
+
+## Logging
+
+In order that application issues are quickly found and diagnosed logs should be easily searchable & discoverable. We should accumulate logs in a single service, with a consistent format.
+
+We should:
+
+- Send all logs to the `logging.wellcomecollection.org` ELK (Elasticsearch, Logstash, Kibana) stack.
+- Log in a consistent format to make the timestamp, originating service and environment easily searchable.
+- Remove all CloudWatch logging.
+
+## Tracing
+
+In order that application issues are quickly found and diagnosed we should be able to follow work occurring across multiple services.
+
+We should:
+
+- Implement a tracing solution in our piplelines that allow us to visualise the flow of work through our services.
+
+## Metrics
+
+In order that application issues are quickly found and diagnosed we should be able to view metrics from our applications easily.
+
+We should:
+
+- Decide on appropriate metric collectors for each of our products.
+- Decide on a single platform for visualising application metrics across our products.
```

Comment:
Is a single platform necessary/realistic? Or could it be a single place from which to access all metric visualising things (possibly residing on different platforms)?